apps/sms/views/inbox/js/inbox.js

```diff
@@ -983,6 +975,8 @@ var InboxView = {
   onDraftSaved: function inbox_onDraftSaved(draft) {
     var threadToUpdate = draft.threadId ? Threads.get(draft.threadId) : draft;
     this.updateThread(threadToUpdate);
+
+    this.showDraftSavedBanner();
```

Comment:
One note: should we show "draft is saved" banner in this case - user edited new message, swiped left to another app (we saved draft) and then moved back to Messages and presses back button? It sounds like the same workflow for the user when he just presses back, but in one case we show banner and in the other case we don't.

Same thing if we save continuously and draft has been already saved before user presses back button.

I'll think about possible solution :)